docs/sphinx/developers/examples/SimpleTiledWriter.java

```diff
@@ -153,19 +153,29 @@ private void cleanup() {
    * To read an image file and write out an OME Tiff tiled image on the command line:
    *
    * $ java SimpleTiledWriter input-file.oib output-file.ome.tiff 256 256
-   * @throws IOException
-   * @throws FormatException
+   * @throws IOException thrown if unable to setup input or output stream for reader or writer
+   * @throws FormatException thrown when setting invalid values in reader or writer 
+   * @throws ServiceException thrown if unable to create OME-XML meta data
+   * @throws DependencyException thrown if failed to create an OMEXMLService
    */
-  public static void main(String[] args) throws FormatException, IOException {
+  public static void main(String[] args) throws FormatException, IOException, DependencyException, ServiceException {
     int tileSizeX = Integer.parseInt(args[2]);
     int tileSizeY = Integer.parseInt(args[3]);
     SimpleTiledWriter tiledWriter = new SimpleTiledWriter(args[0], args[1], tileSizeX, tileSizeY);
 
-    // Read in images from the input and write them out automatically using tiling
-    tiledWriter.readWriteTiles();
-
-    // close the files
-    tiledWriter.cleanup();
+    try {
+      // Read in images from the input and write them out automatically using tiling
```

Comment:
to be done maybe in another PR:
i will split the ``readWriteTiles()`` into 2 methods: ``initialize()`` and ``readWriteTiles()``
to match the other examples